source/atlas-cli-changelog.txt

```diff
@@ -14,6 +14,15 @@
    :depth: 1
    :class: singlecol
 
+.. _atlas-cli-1.15.1:
+
+{+atlas-cli+} 1.15.1
+--------------------
+
+Released 29 February 2024
+
+- Updates internal code. This release doesn't affect {+atlas-cli+} commands.
```

Comment:
seems like we forgot about https://jira.mongodb.org/browse/CLOUDP-233923 and adding release notes to that ticket 

Since we have actual release notes can we replace with something like 
```suggestion
- Fixes an issue where profile configuration may not prompt for the correct organization
```

cc @tibulca

Reply:
@tibulca or @gssbzn updated. tried to provide a bit more detail based on the ticket